### DISCUSSION TITLE
migrate photo cache to external drive in docker

### DISCUSSION THREAD

**Discussion by alexontour** [photoview/photoview#268] [Q&A and Troubleshooting] [answered]
Hello, I am using photoview as a docker container on an Openmediavault NAS (RP4). Unfortunately, I'm running out of space as my SSD is limited to 32 GB. I tried to configure an external path to the cache on the disk in the docker compose file. Unfortunately it does not work.

 - PHOTOVIEW_MEDIA_CACHE = /srv/dev-disk-by-uuid-123456789/ AppData/photoview

volumes:
     - /srv/dev-disk-by-uuid-123456789/AppData/photoview: / api_cache

Any ideas ? Tnak you very much !

Alex

**Comment by viktorstrate** [comment, marked as answer]
`PHOTOVIEW_MEDIA_CACHE` points to the path of the cache _within the docker container_, theirefore you shouldn't change it, keep it as `PHOTOVIEW_MEDIA_CACHE=/api_cache`.

The new volume will automatically map the host directory to the `/api_cache` directory.

**Reply by alexontour** [reply]
Thank you very much. Now it works with this configuration:

....
      - PHOTOVIEW_MEDIA_CACHE=/app/cache
....
    volumes:
      #- api_cache:/app/cache
      - /srv/dev-disk-by-uuid-123456789/AppData/photoview:/app/cache
